doc/man1/openssl-ciphers.pod.in

```diff
@@ -403,6 +403,13 @@ used and only the two suite B compliant cipher suites
 (ECDHE-ECDSA-AES128-GCM-SHA256 and ECDHE-ECDSA-AES256-GCM-SHA384) are
 permissible.
 
+=item B<CBC>
+
+All cipher suites using encryption algorithm in Cipher Block Chaining (CBC)
+mode. These cipher suites are only supported in TLS v1.2. Currently it's alias
```

Comment:
Are they really supported in TLS 1.2?

Reply:
No, it's 1.2 *and earlier*

Reply:
Oh sorry, I mean "up to" instead of "in".